test/gtest/uct/test_md.cc

```diff
@@ -271,6 +271,16 @@ void test_md::dereg_cb(uct_completion_t *comp)
     md_comp->self->m_comp_count++;
 }
 
+bool test_md::is_gpu_ipc() const
+{
+    if ((GetParam().md_name == "cuda_ipc") ||
```

Comment:
shall we instead simply `return (GetParam().md_name == "cuda_ipc") || (GetParam().md_name == "rocm_ipc")`